<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddAuteur</class>
 <widget class="QDialog" name="AddAuteur">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ajouter un auteur</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/main/images/icone.png</normaloff>:/main/images/icone.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Ajouter un nouvel auteur</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label_naissance">
       <property name="text">
        <string>Naissance</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_nom"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_mort">
       <property name="text">
        <string>Mort</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_nom">
       <property name="text">
        <string>Nom complet</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_bio">
       <property name="text">
        <string>Biographie</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_photo">
       <property name="text">
        <string>Photo</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinBox">
       <property name="maximum">
        <number>2222</number>
       </property>
       <property name="value">
        <number>1900</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="spinBox_2">
       <property name="maximum">
        <number>2222</number>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QTextEdit" name="textEdit"/>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_fichier">
         <property name="text">
          <string>Choisir un fichier</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_fichier">
         <property name="text">
          <string>Choisir</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddAuteur</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddAuteur</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>385</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_fichier</sender>
   <signal>clicked()</signal>
   <receiver>AddAuteur</receiver>
   <slot>setImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>342</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>98</x>
     <y>340</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setImage()</slot>
 </slots>
</ui>
